hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -197,12 +204,20 @@ public boolean filesExist(String partition, String instantTime, String... fileId
   public boolean fileExists(String partition, String instantTime, String fileId) {
     try {
       return fs.exists(new Path(Paths.get(basePath, partition,
-          FSUtils.makeDataFileName(instantTime, "1-0-1", fileId)).toString()));
+          getDataFileNameById(instantTime, fileId)).toString()));
     } catch (IOException e) {
       throw new HoodieTestTableException(e);
     }
   }
 
+  public String getDataFileNameById(String fileId) {
```

Comment:
lets use base file consistently in new code . not "data file"

Reply:
ok makes sense. I've renamed the APIs. I do left `org.apache.hudi.common.fs.FSUtils#makeDataFileName(java.lang.String, java.lang.String, java.lang.String)` as is since it is out of test scope. Renaming it would probably go to another PR.